dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsPlanner.cs

```diff
@@ -66,7 +67,18 @@ public Task<HandlebarsPlan> CreatePlanAsync(Kernel kernel, string goal, Cancella
 
     private readonly HandlebarsPlannerOptions _options;
 
-    private HandlebarsPromptTemplateFactory _templateFactory { get; }
+    private readonly HandlebarsPromptTemplateFactory _templateFactory;
+
+    private static readonly PromptExecutionSettings s_defaultPromptExecutionSettings = new()
+    {
+        ExtensionData = new Dictionary<string, object>()
+        {
+            { "temperature", 0.0 },
+            { "top_p", 0.0 },
+            { "presence_penalty", 0.0 },
+            { "frequency_penalty", 0.0 },
+        }
```

Comment:
Can we use `OpenAIPromptExecutionSettings` given this is OpenAI specific? Or if we want to be model agnostic move this elsewhere?

Reply:
Good question. We don't want the Handlebars planner to take a dependency on OpenAI, but when users *do* use OpenAI, we want to use these defaults. Is there a better way to achieve that without taking a dependency on an OpenAI-specific package?

Reply:
I think the best course of action might be to remove this, and just recommend that developers override the model settings when using the planner (especially to turn down the temperature when using OpenAI models). Ultimately it is up to them to tune the settings to what they want, although generally it seems lower temp yields a more reliable planner. I'll make a note of this in the examples.